src/main/kotlin/racingcar/domain/Car.kt

```diff
@@ -0,0 +1,11 @@
+package racingcar.domain
+
+class Car(val id: Int, private val driveCondition: DriveCondition) {
+    var distance: Int = 1
```

Comment:
```suggestion
    var distance: Int = 1
        private set
```

기존 Distansce는 외부에서도 수정 가능한 코드를 가지고 있었습니다. 하지만 private set을 사용하게 된다면 Car 내에서만 수정이 가능하고 Car 외부에서는 수정이 불가능하도록 할 수가 있습니다. 😊

Reply:
코틀린은 set과 get이 자동으로 생성된다는 점이 장점임과 동시에 단점인 것 같네요! 
꼼꼼한 리뷰  감사합니다!